core/modules/user/user.admin.inc

```diff
@@ -710,6 +728,84 @@ function user_admin_role_submit($form, &$form_state) {
   $form_state['redirect'] = 'admin/config/people/roles';
 }
 
+/**
+ * Form to clone a role.
+ *
+ * @ingroup forms
+ * @see user_admin_role_clone_submit()
+ */
+function user_admin_role_clone($form, &$form_state, $role) {
+  backdrop_set_title(t('Clone %source_role role', array('%source_role' => $role->label)), PASS_THROUGH);
```

Comment:
I use `PASS_THROUGH` here, in order to allow the role name to be rendered in italics in the page title. Noting that I've added a `check_plain()` in `user_role_save()` for the role label.